releasenotes/notes/add-qfi-with-primitive-86d935d19dfff1a1.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    New Quantum Fisher Information (QFI) Algorithm using the primitives has been added. It
```

Comment:
Can we add a class ref to  `LinCombQFI` so there is something in the notes the user can click on which will take them to this new class.